Tests failing locally due to missing `py` dependency

Unit tests are failing to run locally due to a missing `py` package:

```sh
$ python -m venv venv
(venv) $ source venv/bin/activate
(venv) $ make deps-posix
(venv) $ make
...
tox -e wheel --notest --installpkg dist/awscli_login-0.2b2.dev25-py3-none-any.whl -vv
ROOT: 140 D setup logging to DEBUG on pid 66813 [tox/report.py:221]
Traceback (most recent call last):
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/bin/tox", line 8, in <module>
    sys.exit(run())
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/run.py", line 19, in run
    result = main(sys.argv[1:] if args is None else args)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/run.py", line 38, in main
    state = setup_state(args)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/run.py", line 53, in setup_state
    options = get_options(*args)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/config/cli/parse.py", line 38, in get_options
    guess_verbosity, log_handler, source = _get_base(args)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/config/cli/parse.py", line 61, in _get_base
    MANAGER.load_plugins(source.path)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/plugin/manager.py", line 90, in load_plugins
    self._register_plugins(inline)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox/plugin/manager.py", line 38, in _register_plugins
    self.manager.load_setuptools_entrypoints(NAME)
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 287, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/setuptools/_vendor/importlib_metadata/__init__.py", line 194, in load
    module = import_module(match.group('module'))
  File "/Users/ddriddle/.pyenv/versions/3.9.13/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/venv/lib/python3.9/site-packages/tox_pyenv.py", line 47, in <module>
    import py
ModuleNotFoundError: No module named 'py'
make: *** [Makefile:59: .tox/wheel/pyvenv.cfg] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -19,10 +19,6 @@ all: test coverage docs doctest
 # Python dependencies needed for local development
 deps: deps-build deps-doc deps-test deps-publish
 
-# Python packages needed to run the tests on a Unix system
-deps-posix: deps
-	$(PIP) tox-pyenv
-
 # Python packages needed to run the tests on a Windows system
 deps-win: deps
 	$(PIP) pyenv-win
@@ -56,7 +52,7 @@ check: .twinecheck
 # Install wheel into tox virtualenv for testing
 install: $(TOX_ENV)
 $(TOX_ENV): build | cache
-	tox -e wheel --notest --installpkg $(WHEEL) -vv
+	tox -e wheel --notest --installpkg $(WHEEL)
 	@touch $@
 
 # Build and save dependencies for reuse

--- a/tox.ini
+++ b/tox.ini
@@ -3,7 +3,7 @@ envlist = py37,py38,py39,py310
 
 [testenv]
 # getpass.getuser fails on Windows if these envs are not passed in
-passenv = LOGNAME USER LNAME USERNAME
+passenv = LOGNAME,USER,LNAME,USERNAME
 # https://packaging.python.org/guides/index-mirrors-and-caches/#caching-with-pip
 install_command=python -m pip install --disable-pip-version-check --find-links=cache {opts} {packages}
 commands = python -m unittest discover -s src {posargs}

